server/src/auth/auth.service.ts

```diff
@@ -17,8 +17,8 @@ import { ContactService } from '../contact/contact.service';
 @Injectable()
```

Comment:
Is this file copy and pasted from old zap-api? There are a lot of documentation, code clarity and logging improvements in the current authService file.